### DISCUSSION TITLE
Sand Raking - Handymen

### DISCUSSION THREAD

**Discussion by Orig-Technofuzz** [OpenRCT2/OpenRCT2#18105] [Feature requests]
While this might be quite difficult to implement (or easy), can you please add a sand raking action in the handymen actions list.  This would also, however, require the addition of a "raked sand tile" which would be similar to the mowed grass tile.

**Comment by 733737** [comment]
Wouldn't it be better to have raked brown dirt instead of raked sand? by the way, this would be very difficult to implement, simply due to the amount of code needed and the dozens of images, especially if you wanted to add a rake to the handyman. You would also need to change replays, network version, and staff AI

**Reply by jjpb44** [reply]
I believe it could be implemented based on the mowing grass action, just replacing the parameter (grass) with the sand and instead of mowed grass put the indicated raked sand tile.
